### ISSUE TITLE
Multiplayer no response

### ISSUE THREAD

**Issue by Zeuscp1** [tsunamayo/Skywanderers#1217]
### Short Description of the issue:

...The game won't try to load or give you a ''connection failed'' message, just simply wont try to do anything when you click join server. I don't know if this is normal

### Steps to reproduce the issue:

...Port fowarded friend , try to join the friend

### Environment
- Skywanderers version: lastest
- Graphics Card: GTX 1080ti

**Comment by tsunamayo** [comment]
Yes multiplayer is barebone, and I will have to migrate the code as Unity is deprecating all the feature I used. Did your friend opened his game to other? Thanks

**Comment by Zeuscp1** [comment]
We both tried connecting each other via port forward and 3rd party programs like hamachi.
It doesn't matter what we do we could't connect because when you press the ''play'' button nothing happens, it doesn't try to connect or gives you a fail or success message like the button is not linked to any action i don't know if i explain myself correctly

**Comment by tsunamayo** [comment]
Yes if it fails it wont tell you why. Did the host went into the pause menu and clicked "Open to multiplayer ?" Thanks

**Comment by Zeuscp1** [comment]
Yes we tried with both open and close multiplayer and none of them worked, if it not too much to ask me and some more people would like multiplayer fixed as it would add a great amount of ''gameplay'' the posibility of having friends in our world helping us build ships and do stuff with us will be a lot of fun and is something we can have to ''wait'' until more stuff is added. No rush tho :D thank you for your hard work

**Comment by RougeSable** [comment]
Hi @tsunamayo I've the same behavior as of today 😬 Without any error on the client game, it's really hard to guess what happen unfortunately
